# HG changeset patch
# User Jaroslav Hajek <highegg@gmail.com>
# Date 1223022111 -7200
#      Fri Oct 03 10:21:51 2008 +0200
# Branch release-3-0-x
# Node ID 9026c7cc22effa8c6694bbdf0ca3ebc55c6f08ac
# Parent  5e7a180a0babe6fe198c0341cf029e96607956cd
Added tag release-3-0-3 for changeset 152a68291b66

diff --git a/.hgtags b/.hgtags
--- a/.hgtags
+++ b/.hgtags
@@ -32,9 +32,10 @@ afbd31bb7b4e001c6c16d41eb908aa66bed64352
 fc55a5e1760b832a672d9de3b53888a3d6793d21 ss-2-9-14
 0b91144f9533b05d14fb5692ed67d8c6f500230f ss-2-9-15
 5608935ff5228a90d34fa80dce3dab2462d20fd3 ss-2-9-16
 77de8319c337b39a477d5433348e4597ad61abe0 ss-2-9-17
 9f02c32eb70d3e6d0c42a2fd5f3b862f4d6c6bcc ss-2-9-18
 89d546610556ee47ec634ab8c3e441e1db248873 ss-2-9-19
 57504aa0461e08a1771f3c66f0e6b6759d564887 release-3-0-1
 928193724f3b72bfc8a1a62e81ddb9bde9e63f5e release-3-0-2
+152a68291b66b43095a0998870ad09200472d321 release-3-0-3
 274d9642ac1032fcc4996a3cac80cfe5fd20f8a4 release-3-0-0
